Firmware/Marlin_main.cpp

```diff
@@ -7238,6 +7238,87 @@ SERIAL_PROTOCOLPGM("\n\n");
 
     }break;
     #endif // FWRETRACT
+    /*!
+    ### M214 - Set Arc configuration values (Use M500 to store in eeprom)
+
+    #### Usage
+
+        M214 [N] [S] [R] [F]
+
+    #### Parameters
+    - `N` - A float representing the max and default millimeters per arc segment.  Must be greater than 0.
+    - `S` - A float representing the minimum allowable millimeters per arc segment.  Set to 0 to disable
+    - `R` - An int representing the minimum number of segments per arcs of any radius,
+            except when the results in segment lengths greater than or less than the minimum
+            and maximum segment length.  Set to 0 to disable.
+    - 'F' - An int representing the number of segments per second, unless this results in segment lengths
+            greater than or less than the minimum and maximum segment length.  Set to 0 to disable.
+    */
+    case 214: //!@n M214 - Set Arc Parameters (Use M500 to store in eeprom) N<MM_PER_ARC_SEGMENT> S<MIN_MM_PER_ARC_SEGMENT> R<MIN_ARC_SEGMENTS> F<ARC_SEGMENTS_PER_SEC>
+    {
+        // Extract N
+        float n = cs.mm_per_arc_segment;
+        float s = cs.min_mm_per_arc_segment;
+        int r = cs.min_arc_segments;
+        int f = cs.arc_segments_per_sec;
+        // Extract N
+        if (code_seen('N'))
```

Comment:
Please avoid using N. It has special meaning when printing from USB. It might not generate a conflict, but let's stay on the safe side :)

Reply:
Absolutely fine with me.  Any suggestions as to what I should use instead of N?

Reply:
You can use P, W or any other letter that doesn't have a special meaning somewhere else. I recommend that you use P